src/api/tasks.js

```diff
@@ -1,14 +1,14 @@
 
 import Q from "q";
 import logger from "winston";
-import { Task } from "../model/tasks";
-import { Transaction } from "../model/transactions";
-import { AutoRetry } from "../model/autoRetry";
+import { TaskModelAPI } from "../model/tasks";
+import { TransactionModelAPI } from "../model/transactions";
+import { AutoRetryModelAPI } from "../model/autoRetry";
 import * as Channels from "../model/channels";
```

Comment:
@BMartinos This file is using an exported function 'isChannelEnabled' (line 98); we can import individually both ChannelModel and isChannelEnabled. I am not sure however if importing individual function is a pattern we use.